WebServer/microservices/db/data.sql

```diff
@@ -11,8 +11,6 @@ SET xmloption = content;
 SET client_min_messages = warning;
 SET row_security = off;
 
-CREATE DATABASE traffic_annonymisation_db;
```

Comment:
Hvorfor er den her linje slettet?

Reply:
POSTGRES_DB: This optional environment variable can be used to define a different name for the default database that is created when the image is first started. (https://hub.docker.com/_/postgres)
Når environment variablen er sat, laver postgres containeren automatisk sådan en database. Dermed får man en fejl, når postgres prøver at køre denne linje. Det er tilfældet både for Docker og Singularity, som bruger samme Docker image